runtime/compiler/x/runtime/X86PicBuilder.nasm

```diff
@@ -371,7 +363,7 @@ mergePopulateIPicClass:
       LoadClassPointerFromObjectHeader eax, edx, edx              ; receiver class
       mov         eax, dword  [edx+J9TR_J9Class_classLoader]      ; receivers class loader
 
-      mov         ebx, dword  [esi+eq_IPicData_interfaceClass]    ; ebx == IClass from IPic data area
+      mov         ebx, dword  [esi+eq_IPicData_interfaceClass]    
```

Comment:
Why is that comment removed?  Seems still relevant to me.

Reply:
That commit in the .pasm file was the one that introduced the esi offset, and then deleted the comment.  For some reason your pnasm file still had the comment along with the new code with the offset (it was in some hybrid state).   That's OK--I think you can leave the code as-is.